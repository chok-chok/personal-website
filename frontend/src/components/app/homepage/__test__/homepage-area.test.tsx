import * as React from 'react';
import Homepage from '../homepage-area';
import { shallow } from 'enzyme';

describe('Test Homepage component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<Homepage />);
    expect(tree).toMatchSnapshot();
  });
});
