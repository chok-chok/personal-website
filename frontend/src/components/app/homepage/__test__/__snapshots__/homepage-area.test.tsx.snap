// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Homepage component should renders correctly 1`] = `
<styled__HomepageAreaBackground>
  <styled__HomepageIntroSectionWrapper>
    <Intro />
  </styled__HomepageIntroSectionWrapper>
  <styled__HomepageUpdateSectionWrapper />
</styled__HomepageAreaBackground>
`;
